packages/gatsby-remark-katex/README.md

```diff
@@ -16,7 +16,11 @@ plugins: [
     resolve: `gatsby-transformer-remark`,
     options: {
       plugins: [
-        `gatsby-remark-katex`,
+        resolve: `gatsby-remark-katex`,
+        options: {
+          // Add any KaTeX options from https://github.com/KaTeX/KaTeX/blob/master/docs/options.md here
+          strict: `ignore`
+        }
```

Comment:
we are missing `{}` here to wrap `resolve` and `options` fields

Reply:
Ah, sorry! Have become incapable of writing code with Prettier 😛

Reply:
I know the feeling 👍